Arduboy.cpp

```diff
@@ -682,14 +687,12 @@ unsigned char* Arduboy::getBuffer()
 
 boolean Arduboy::pressed(uint8_t buttons)
 {
- uint8_t button_state = getInput();
- return (button_state & buttons) == buttons;
```

Comment:
Please change it back.  I wrote it that way on purpose.  Though honestly my desire for it probably reflects on the poorly named getInput() more than anything else (because I don't disagree with your logic).

What about `pollButtonState()`?